docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_6.md

```diff
@@ -0,0 +1,31 @@
+# Audit of partially sharing storage Data access 
+
+Test verifies that any access to the partially sharing in the object storages data is logged.
+
+**Prerequisites**:
+- Admin user
+- Non-admin user
+
+**Preparations**:
+1. Login as the admin user from the prerequisites 
+2. Create storage `storage1` and give only ***READ*** permission for non-admin user from the prerequisites
+3. In the `storage1`
+    - Add `file1` and `file2` to the storage root
+    - Create `folder1` with 2 files (`inner_file1` and `inner_file2`)
+    - Create `folder2` with 2 files (`inner_file3` and `inner_file4`)
+
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Open the **Library** page | |
+| 2 | Open the object storage `storage3` created at step 2 of the Preparations | |
+| 3 | Mark checkboxes near `file1` and `folder1` | |
+| 4 | Click ***Share 2 items*** button | Pop up ***Select users / groups to share 2 items*** appears |
+| 5 | Add non-admin user to list of selected user | |
+| 6 | Give **Read** permissions | |
+| 7 | Click **SAVE AND SHARE** button | Pop up with Link to 2 shared items appears |
+| 8 | Copy share link to clipboard | |
+| 9 | Logout | |
+| 10 | Open link stored at the step 5 of the Preparations. Login as the non-admin user from the prerequisites | The page with the header ***storage1-share-1*** appears that contains only `file1` and `folder1` created at step 3 of the Preparations |
+| 11 | Click **Download** button near the `file1` name | |
+| 12 | Logout and _Check audit log_ (see below) | Record <br> `READ storage1/file1` <br> is shown in the top of logs records |
```

Comment:
Add the section **After** in which the storage created at the Preparations shall be removed.

Reply:
Done